oxidation/libparsec/crates/crypto/Cargo.toml

```diff
@@ -15,17 +15,25 @@ base32 = "0.4.0"
 
 sodiumoxide = { version = "0.2.7", optional = true }
 libsodium-sys = { version = "0.2.7", optional = true }
+openssl = { version = "0.10", optional = true, features = ["vendored"] }
+openssl-sys = { version = "0.9", optional = true }
```

Comment:
`openssl` already contains dependency on `openssl-sys` (and it specify the specific version to use given the two must be closely kept in sync), it's better to only refere to `openssl` here